src/domain/app.rs

```diff
@@ -48,18 +47,25 @@ where
 {
     app.start_ui()?;
 
+    let mut event_server = event::CrosstermEventServer::default();
```

Comment:
Could use an "Event Server Factory" potentially instead on the `EventServer` abstraction 🤷